content/ru/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Финализаторы
+content_type: concept
+weight: 80
+---
+
+# Финализаторы
+
+Финализаторы — это ключи в пространстве имен, которые сообщают Kubernetes дождаться выполнения определенных условий, прежде чем он полностью удалит ресурсы, помеченные для удаления. Финализаторы предупреждают контроллеры о необходимости очистить ресурсы, принадлежащие удаленному объекту.
+
+Когда вы указываете Kubernetes удалить объект, для которого указаны финализаторы, API Kubernetes помечает объект для удаления, заполняя .metadata.deletionTimestamp, и возвращает код состояния 202 (HTTP «Принято»). Целевой объект остается в завершающем состоянии, пока плоскость управления или другие компоненты выполняют действия, определенные финализаторами. После завершения этих действий контроллер удаляет соответствующие финализаторы из целевого объекта. Если поле Metadata.finalizers пусто, Kubernetes считает удаление завершенным и удаляет объект.
+
+Вы можете использовать финализаторы для управления сборкой мусора ресурсов. Например, вы можете определить финализатор для очистки связанных ресурсов или инфраструктуры перед тем, как контроллер удалит целевой ресурс.
```

Comment:
In the English language, using repeated "you" is fine, but we would prefer to avoid these repeated words (e.g., making some phrases impersonal) in Russian.
```suggestion
Вы можете использовать финализаторы для управления сборкой мусора ресурсов. Например, можно определить финализатор для очистки связанных ресурсов или инфраструктуры перед тем, как контроллер удалит целевой ресурс.
```